tsconfig.json

```diff
@@ -22,7 +22,7 @@
     "skipLibCheck": true,
     "strict": true,
     "target": "esnext",
-    "types": ["react-native", "jest"]
+    "types": ["react-native", "jest", "@testing-library/jest-native"]
```

Comment:
Are you sure we need this ? It looks like this lib provides typings and suggests another way to use types

![Screenshot_2023-09-30-17-27-13-25_3aea4af51f236e4932235fdada7d1643.jpg](https://github.com/BearStudio/react-native-ficus-ui/assets/3920615/6d8e6c12-e5ed-408f-b20d-0df355b4602a)

Reply:
Without it, the build action will fail https://github.com/BearStudio/react-native-ficus-ui/actions/runs/6362862113/job/17278149636
This is the only way I have found to fix it